tensorflow/contrib/learn/python/learn/learn_io/data_feeder.py

```diff
@@ -106,7 +106,7 @@ def setup_train_data_feeder(
   Args:
     x: numpy, pandas or Dask matrix or dictionary of aforementioned. Also supports iterables.
     y: numpy, pandas or Dask array or dictionary of aforementioned. Also supports iterables.
-    n_classes: number of classes. Must be None or same type as y. In case, y is dict ( or iterable returns dic)
+    n_classes: number of classes. Must be None or same type as y. In case, y is dict (or iterable returns dic)
       n_classes[key] = n_classes for y[key]
```

Comment:
Can you put in backticks every expression that's code or pseudocode? See the documentation howto. The docstrings will be rendered as markdown, so in this case, say `n_classes[key] = n_classes for y[key]`, not n_classes[key] = n_classes for y[key].